src/py21cmmc/likelihood.py

```diff
@@ -792,7 +1032,7 @@ class LikelihoodNeutralFraction(LikelihoodBase):
     and 0 otherwise.
     """
 
-    required_cores = ((core.CoreLightConeModule, core.CoreCoevalModule),)
+    required_cores = ((core.CoreLightConeModule, core.CoreCoevalModule), (core.CoreCMB))
```

Comment:
I think this will _enforce_ that `core.CoreCMB` is loaded, which is probably not what you want?

Reply:
What I wanted to include is that we must have either one of these three options: `core.CoreLightConeModule`, `core.CoreCoevalModule`, `core.CoreCMB`. I can't remember why I put `core.CoreCMB` outside `(core.CoreLightConeModule, core.CoreCoevalModule)`. But why did we use `((core.CoreLightConeModule, core.CoreCoevalModule),)` rather than `(core.CoreLightConeModule, core.CoreCoevalModule)` in the first place?

Reply:
Having them inside a tuple inside the tuple means that you can have _either_ `CoreLightConeModule` or `CoreCoevalModule` to make the likelihood work. Having them like
`(core.CoreLightConeModule, core.CoreCoevalModule)` means you need _both_ to be loaded. Essentially, top-level of the tuple is AND, and second-level tuples are OR.